templates/addons/etcd-backup-restore/sse-key-secret.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: v1
+kind: Secret
+type: addons.cluster.x-k8s.io/resource-set
+metadata:
+  name: ${CLUSTER_NAME}-ssekey
+stringData:
+  linode-token-region.yaml: |-
```

Comment:
can we rename this to reflect the different secret

Reply:
done